### ISSUE TITLE
Make it possible to disable search through appsettings

### ISSUE THREAD

**Issue by Stefan-Truelime** [Kentico/xperience-by-kentico-azure-ai-search#12]
**Is your feature request related to a problem? Please describe.**
When upgrading to to kentico 29.0.0 we encountered some issues with the azure ai search package. #11 

**Describe the solution you'd like**
It would be nice if we can enable/disable the azure ai search in the appsettings.json like this

```json
  "CMSAzureSearch": {
    "SearchServiceEnabled" : false //default would be true
    "SearchServiceEndPoint": "https://my-endpoint.net",
    "SearchServiceAdminApiKey": "my-admin-api-key",
    "SearchServiceQueryApiKey": "my-query-api-key"
  }
```

**Describe alternatives you've considered**
for now clearing out the SearchServiceEndPoint also 'works' i'm now able to publish pages again, but eventlog is flowing with errors

**Additional context**
in case for any future issue, we can just disable search and continue with our work

**Comment by liparova** [comment]
Thank you for bringing this to our attention. We'll examine it internally and keep you updated on our progress.

**Comment by michalJakubis** [comment]
Hi @Stefan-Truelime, fix has been released in [v3.0.0](https://github.com/Kentico/xperience-by-kentico-azure-ai-search/releases/tag/v3.0.0)
